<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShiftTimesDialog</class>
 <widget class="QDialog" name="ShiftTimesDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>224</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Shift selected dive times</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/subsurface-icon</normalon>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item alignment="Qt::AlignTop">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Shift times of selected dives by</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="shiftedTimeLabel">
          <property name="text">
           <string>Shifted time:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="currentTimeLabel">
          <property name="text">
           <string>Current time:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="currentTime">
          <property name="text">
           <string>0:0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="shiftedTime">
          <property name="text">
           <string>0:0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEdit">
        <property name="date">
         <date>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="maximumDate">
         <date>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="displayFormat">
         <string>h:mm</string>
        </property>
        <property name="timeSpec">
         <enum>Qt::LocalTime</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="backwards">
        <property name="text">
         <string>Earlier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="forward">
        <property name="text">
         <string>Later</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../subsurface.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ShiftTimesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ShiftTimesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeEdit</sender>
   <signal>timeChanged(const QTime)</signal>
   <receiver>ShiftTimesDialog</receiver>
   <slot>changeTime()</slot>
  </connection>
  <connection>
   <sender>backwards</sender>
   <signal>toggled(bool)</signal>
   <receiver>ShiftTimesDialog</receiver>
   <slot>changeTime()</slot>
  </connection>
 </connections>
</ui>
